generated/ApiResponse.h

```diff
@@ -0,0 +1,46 @@
+#ifndef ORG_OPENAPITOOLS_MODEL_ApiResponse_H_
+#define ORG_OPENAPITOOLS_MODEL_ApiResponse_H_
+
+#include "serialization.h"
+
+#include <optional>
+
+#include <string>
+
+namespace org {
+namespace openapitools {
+namespace model {
+
+/**
+ * \brief Describes the result of uploading an image resource
+ */
+struct ApiResponse {
+    static const char *dtoName() { return "swagger::dto::ApiResponse"; }
+
+    // required fields
+
+    // optional fields
+    std::optional<int32_t> code;
+    std::optional<std::string> type;
+    std::optional<std::string> message;
+};
+
+void to_json(nlohmann::json& j, const ApiResponse& obj) {
+    j = nlohmann::json{
+        { "code", obj.code },
+        { "type", obj.type },
+        { "message", obj.message },
+    };
+}
+
+void from_json(const nlohmann::json& j, ApiResponse& obj) {
+    j.at("code").get_to(obj.code);
+    j.at("type").get_to(obj.type);
+    j.at("message").get_to(obj.message);
```

Comment:
does this work if `"message"` is missing?

Reply:
Oof... no it doesn't :man_facepalming:

Reply:
(Since the conversation is gone from the files tab) I introduced `to_optional` to address this. If we `to_json ∘ from_json` now, the missing field will be `null`. Here's a pet, where `id` was missing from the raw json string:
```
{"category":{"id":1,"name":"non-food"},"id":null,"name":"doggie","photoUrls":["https://doggie.pics/0"],"status":"available","tags":[{"id":0,"name":"myTag"}]}
```